docs/feature_encoders.md

```diff
@@ -67,6 +67,8 @@ Additionally, if one side does not have an encoder, you can specify `{}` for the
 #define ENCODER_RESOLUTIONS_RIGHT { 4 }
 ```
 
+Keep in mind that whenver you change the encoder resolution, you will need to reflash the half that has the encoder affected by the change.
```

Comment:
```suggestion
!> Keep in mind that whenver you change the encoder resolution, you will need to reflash the half that has the encoder affected by the change.
```

This renders it in a box that is a bit more ... urgent. And you can run `qmk docs` to serve the site locally to see the changes.